tests/MercadoPago/Client/Invoice/InvoiceClientUnitTest.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace MercadoPago\Client\Invoice;
```

Comment:
IMO, the classes for the test suite should have a different namespace, with a separate PSR-4 autoload
https://github.com/mercadopago/sdk-php/blob/5d77e6ba9319392c1989183e591d43c865883bae/composer.json#L11

Reply:
Is the idea to do something like this [example](https://getcomposer.org/doc/04-schema.md#:~:text=autoload%2Ddev%20section.-,Example,-%3A)?
I mean, having another level indicating it's a test folder, doing something like the example below? Can you give me an example?

```json
    "autoload": {
        "psr-4": {
            "MercadoPago\\": "src/MercadoPago",
            "Tests\\MercadoPago\\": "tests/MercadoPago"
        }
    },
```

Reply:
Near. It's more like:
```json
"autoload": {
    "psr-4": {
        "MercadoPago\\": "src/MercadoPago/"       
    }
},
"autoload-dev": {
    "psr-4": {
        "MercadoPago\\Tests\\": "tests/MercadoPago/"
    }
}
```

Reply:
https://getcomposer.org/doc/04-schema.md#autoload-dev.

Reply:
Cool, seems like a better way to do it. I'm not going to make this modification to this PR so as not to pollute the PR, since I'll have to adjust other places in the code. When I do, I'll put you as a reviewer if you don't mind.
Thank you very much for the suggestion, it helps a lot for us to make a better code!